src/components/other/ExportPreview/index.js

```diff
@@ -125,10 +125,11 @@ export default class ExportPreview extends React.PureComponent {
                 <GalleryViewer
                     url={exportObj.file}
                     mimeType={exportObj.mimeType}
-                    // NOTE: should not this be true
+                    // NOTE: should not this be true?
                     canShowIframe={false}
```

Comment:
@thenav56  `canShowIframe` for documents generated in deep should be `true`, right?

Reply:
Yeah